### ISSUE TITLE
Select dropdown caret not visible in Windows High Contrast Mode

### ISSUE THREAD

**Issue by Stanton** [jadu/pulsar#959]
**Describe the bug**
The dropdown arrow on select/select2 elements is not visible when Windows is using high contrast mode.

This reduces the overall affordance of the control and reduces the accessibility of it.

**To Reproduce**
1. Go to Settings > Ease of Access > Theme settings > High contrast settings = on

**Expected behavior**
The dropdown caret to be clearly visible on the right hand side of the select/select2 element.

![Screenshot 2019-03-21 at 15 48 15](https://user-images.githubusercontent.com/18653/54765076-c7554800-4bf0-11e9-95e9-1de99101dfdd.png)

**Screenshots**
Screenshot of Lexicon:
![Screenshot 2019-03-21 at 15 42 21](https://user-images.githubusercontent.com/18653/54764772-4d24c380-4bf0-11e9-9838-9161379011dc.png)

Close up view of select & select2:
![Screenshot 2019-03-21 at 15 42 08](https://user-images.githubusercontent.com/18653/54764839-66c60b00-4bf0-11e9-9dcb-e4a475dd1a46.png)

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Browser: Edge
